ambry-frontend/src/test/java/com.github.ambry.frontend/AmbryBlobStorageServiceTest.java

```diff
@@ -1325,36 +1323,34 @@ public void close()
    * Enumerates the different operation types in the router.
    */
   enum OpType {
-    DeleteBlob,
-    GetBlobInfo,
-    GetBlob,
-    PutBlob
+    DeleteBlob, GetBlob, PutBlob
```

Comment:
any reason to move them to a single line? This is ok with our coding style conventions?